### PR TITLE
special test theme for visual tests

### PR DESCRIPTION
**Pull request by toaster**
### Description:

This PR introduces a new, less ugly test theme for usage with visual tests.

### Checklist:

- [ ] Tests included.
- [x] Lint and formatter run with no errors.
- [x] Tests all pass.

### CODE REVIEW

File: widget/hyperlink.go [hunk 1]
```diff
@@ -94,7 +94,7 @@ func (hl *Hyperlink) textStyle() fyne.TextStyle {
 
 // textColor tells the rendering textProvider our color
 func (hl *Hyperlink) textColor() color.Color {
-	return theme.HyperlinkColor()
```
Comment (andydotxyz) [inline widget/hyperlink.go:97]: This is going to break custom themes.
We deprecated the Hyperlink color for a reason, we can't really change it until 2.0

**Review by andydotxyz — CHANGES_REQUESTED** [review]
Adding a new ttheme to do this makes sense - but I don't think we can change widget behaviour to do this.
Just because the builtin theme makes HyperlinkColor the same as Primary colour does not mean that custom themes will have done the same.

File: theme/icons.go [hunk 2]
```diff
@@ -143,7 +143,7 @@ func (res *DisabledResource) Name() string {
 
 // Content returns the disabled style content of the correct resource for the current theme
 func (res *DisabledResource) Content() []byte {
-	clr := fyne.CurrentApp().Settings().Theme().DisabledIconColor()
```
Comment (andydotxyz) [inline theme/icons.go:146]: Oops, same as hyperlink, dont change the public API use of colour yet

File: widget/hyperlink.go (hunk 1 above)
Comment (toaster) [reply widget/hyperlink.go:97]: Okay, then I will remove the deprecated color from the test theme and apply the same delegation as for the internal themes.

**Pull request closed without merging**